src/backend/overviewSite/index.html

```diff
@@ -174,140 +174,112 @@ <h1>Open Event Web Apps</h1>
     <h2 class="margin"> Apps with expandable sessions page </h2>
     <div class="row text-left">
         <div class="col-md-4 col-xs-12 col-sm-6 event-holder">
-            <div class="thumbnail">
-                <a href='./FOSSASIASummit/index.html' target='_blank'>
-                    <img src='./fasmall.jpg' onerror="imgError(this, true);">
-                </a>
+            <div class="thumbnail" onclick="window.open('./FOSSASIASummit/index.html')">
+                <img src='./fasmall.jpg' onerror="imgError(this, true);">
                 <div class="caption">
                     <h4 class="name">FOSSASIA Summit 2018</h4>
                 </div>
             </div>
         </div>
         <div class="col-md-4 col-xs-12 col-sm-6 event-holder">
-            <div class="thumbnail">
-                <a href='./OpenTechSummit/index.html' target='_blank'>
+            <div class="thumbnail" onclick="window.open('./OpenTechSummit/index.html')">
```

Comment:
There's absolutely nothing wrong with the anchor element. There's no need to create an onclick listener to open a new webpage when this can be achieved without javascript, and without popups.

What you should do instead to fix the problem is to to extend the anchor link to include the caption.

```html
 <a href='./OpenTechSummit/index.html' target='_blank'>	
                 <img src='./otssmall.jpg' onerror="imgError(this, true);">	                 <img src='./otssmall.jpg' onerror="imgError(this, true);">
                
                 <div class="caption">	                 <div class="caption">
                    <h4 class="name">Open Tech Summit 2017</h4>	+                    <h4 class="name" onclick="window.location.href=''">Open Tech Summit 2017</h4>
                 </div>
</a>
```

Reply:
I tried doing this, but it disrupted the design. No worries I will find another way to do it.